pypeit/spectrographs/vlt_fors.py

```diff
@@ -301,17 +317,23 @@ def config_specific_par(self, scifile, inp_par=None):
         #self.set_detector(detector)
         # Wavelengths
         #par['calibrations']['wavelengths']['nonlinear_counts'] = self.detector[0]['nonlinear'] * self.detector[0]['saturation']
-
         if self.get_meta_value(scifile, 'dispname') == 'GRIS_300I':
             par['calibrations']['wavelengths']['reid_arxiv'] = 'vlt_fors2_300I.fits'
             par['calibrations']['wavelengths']['method'] = 'full_template'
         elif self.get_meta_value(scifile, 'dispname') == 'GRIS_300V':
             par['calibrations']['wavelengths']['reid_arxiv'] = 'vlt_fors2_300V.fits'
             par['calibrations']['wavelengths']['method'] = 'full_template'
-
-        if 'lSlit' in self.get_meta_value(scifile, 'decker'):
+        elif self.get_meta_value(scifile, 'dispname') == 'GRIS_600z':
```

Comment:
Do we have this grating in the DevSuite?  If not, add it.

Reply:
This will be added in a dev suite PR that will follow the objfind_refactor dev suite PR.